.travis.yml

```diff
@@ -10,14 +12,6 @@ env:
    #   via the "travis encrypt" command using the project repo's public key
    - secure: "MAirt38nl74gOmupg6oWoghoXT8YDgjMJmPzrv6YcMLJ92tS8kv7PGtSfdM+BlxXwlvvXIlqXNwb2R+U4CnjsZUsL+28RGqFiKReAbDUON55Cucb/Yy0ZB+QC9NpCxM5LfC4BycDPI6HStx7ZmDy61/U9alogrbhiqomD/LlMTQ="
 
-addons:
-  coverity_scan:
-    project:
-      name: "BorderTech/wcomponents"
-      description: "Build submitted via Travis CI"
-    notification_email: jonoa4n@gmail.com
-    build_command_prepend: "mvn clean"
-    build_command: "mvn -DskipTests compile"
-    branch_pattern: coverity_scan
-
-script: "mvn verify"
+## Travis installs the project with the following maven command:- "mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V"
```

Comment:
This comment is certainly essential. Until I read it I was wondering where the `install` phase was.